R/zoop_downloader.R

```diff
@@ -183,10 +195,135 @@ Zoopdownloader <- function(
       dplyr::group_by(dplyr::across(-.data$CPUE))%>%
       dplyr::summarise(CPUE=sum(.data$CPUE, na.rm=TRUE))%>% #Some taxa now have the same names (e.g., CYCJUV and OTHCYCJUV) so we now add those categories together.
       dplyr::ungroup()%>%
+      dplyr::left_join(stations, by=c("Source", "Station"))%>% #Add lat and long
```

Comment:
This is very nitpicky, but can you move this `left_join` to after the next line (`as_tibble`) so it's the final line of this pipe? I just get a little worried about doing the join within the `dtplyr` operation since `dyplyr` can be buggy and so I usually just use it for `group_by` and `summarise`

Reply:
Sure thing